CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 * Unknown `--except` rules will now emit a warning instead of being silently ignored ([#94](https://github.com/stjude-rust-labs/sprocket/pull/94))
 
+### Added
+
+* Added configuration file support ([#104](https://github.com/stjude-rust-labs/sprocket/pull/104)).
```

Comment:
I'd add some more here. Considering that we're going to use the CHANGELOG as the basis for release notes in the future, I'm ok being verbose. I'd mention the new `config` subcommand at least

Reply:
I was trying to figure out where else to add documentation. Currently the sprocket documentation is in rough shape.

Reply:
Yea I see what you mean. sprocket.bio is lacking much about the actual `sprocket` bin.

@claymcleod before this next release of Sprocket (with `run`), we probably want to add some more to the docs site.

Reply:
@a-frantz - I added a markdown page to the docs with information. It's currently a minimalist document.